### DISCUSSION TITLE
Cost Estimation (Inssurance)

### DISCUSSION THREAD

**Discussion by chaimt** [ExperienceLovelace/ha-floorplan#119] [Q&A] [answered]
Hi,
   Don't know if you agree, but I need to renew my house inssurane, and need a cost estimation of my house content.
  If somehow we could add to the room an list of possetions with cost, it would help a lot for the final and upkeep of the cost estimation

**Comment by exetico** [comment, marked as answer]
I can't see why that should be related to floorplan? 🤔 That's more like a Excel-sheet, and a few hours of work.

**Reply by exetico** [reply]
Marked this as the answer. I can't really see why this should go into floorplan. Maybe you've misunderstand the purpose  :)
